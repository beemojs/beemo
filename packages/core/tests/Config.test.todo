// import Config from '../src/Config';

// describe('Config', () => {
//   let config: Config;

//   beforeEach(() => {
//     config = new Config('beemo');
//   });

//   it('errors if no module', () => {
//     expect(() => {
//       config.configure({ module: '' });
//     }).toThrowErrorMatchingSnapshot();
//   });

//   it('doesnt error if module is defined with env var', () => {
//     process.env.BEEMO_CONFIG_MODULE = 'test-boost';

//     let opts: { module?: string } = {};

//     expect(() => {
//       opts = config.configure({});
//     }).not.toThrow();

//     expect(opts.module).toBe('test-boost');

//     process.env.BEEMO_CONFIG_MODULE = '';
//   });
// });
