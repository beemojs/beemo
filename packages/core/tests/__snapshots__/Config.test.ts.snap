// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config errors if no module 1`] = `
"The following validations have failed for \`Config\`:
  - Invalid field \\"module\\". String cannot be empty."
`;
